istio-cni/templates/daemonset.yaml

```diff
@@ -26,7 +26,8 @@ spec:
         # marks the pod as a critical add-on, ensuring it gets
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
-        scheduler.alpha.kubernetes.io/critical-pod: ''
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+        sidecar.istio.io/inject: "false"
```

Comment:
did we want to add this? it seems right but not sure why we don't have it before? just because it was in Istio system which was ignored by injector?

Reply:
OK, Yes, the annotation is only used by `isio-sidecar-injector`, in `CNI` mode, we don't need that.